.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+- '0.12'
+sudo: false
```

Comment:
I don't think we need `sudo: false` anymore, Travis now [uses it by default](http://docs.travis-ci.com/user/workers/container-based-infrastructure/).